portal/app/core/assets/css/d3js.css

```diff
@@ -0,0 +1,56 @@
+svg.simple-diagram {
+font-family: Arial, Helvetica, sans-serif;
```

Comment:
Why are there no indentations in this file?
Did you write this or this from somewhere else?

Reply:
I cleaned up this file